### DISCUSSION TITLE
ability to disable the dismantle option

### DISCUSSION THREAD

**Discussion by knetterpiet** [runelite/runelite#15756] [Ideas]
I would love to get the option to disable the option to dismantle certain items before my group loses it.

i know there is a plugin that removes the examine so i'd like something similar to that.

**Comment by Alexsuperfly** [comment]
the [Custom Menu Swaps plugin](https://runelite.net/plugin-hub/show/hotkeyable-menu-swaps) on the plugin hub can do this as far as i know.

**Reply by knetterpiet** [reply]
i might have to look around in there then. 
thanks for the quick reply
